script.js

```diff
@@ -0,0 +1,402 @@
+const hamburgerMenu = document.getElementById('hamNenu');
```

Comment:
- [ ] You have done a good job with the popup window functionality :100: but there is an issue because only you show the mobile version popup also for the desktop version. And according to the Figma design, these should be different designs. Please, address this issue in order to match the original design :wink: :raised_hands:

| Desktop version | Mobile version |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147254743-c8fdb82c-75d3-4b30-9164-d5ba6f82084c.png) | ![image](https://user-images.githubusercontent.com/81522435/147254778-9f3a416e-7250-46b5-a97f-9c0ac87eec55.png) |

- [ ] Also, only the first project makes the popup appear, and I think that the correct behavior should be that all the projects should make the popup appear. Please, kindly make the popup appear when clicking any project :smiley:

Reply:
Done. Thanks